lib/validate_style.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var validateConstants = require('./validate/validate_constants');
+var validate = require('./validate/validate');
+var latestStyleSpec = require('../reference/latest.min');
+var parseStyle = require('./parse_style');
```

Comment:
Ah. I was thinking we included `jsonlint` in mapbox-gl-js (though, it seems pretty obvious that we wouldn't in retrospect). I will fix the situation.